### PR TITLE
Add live reload instructions to the Readme

### PR DESCRIPTION
**Pull request by mehdi-vaadin**
Fix https://github.com/vaadin/flow/issues/7742

This change is [Reviewable](https://reviewable.io/reviews/vaadin/skeleton-starter-flow-spring/318)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ If you want to run the application locally in the production mode, run `mvn spri
 
 To run Integration Tests, execute `mvn verify -Pintegration-tests`.
 
+### Live Reload (optional)
+
+When you change your Java code and recompile it, the application will be automatically reloaded without need to restart it.
```
Comment (samie) [inline README.md]: Already good as-is, but a suggestions to explain the main idea: 

With live reload you can see results of your code changes immediately. When you edit your Java code and recompile it, the application changes will be automatically reloaded and browser is refreshed.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,20 @@ If you want to run the application locally in the production mode, run `mvn spri
...
+This is done by leveraging [Spring Boot Developer Tools](https://docs.spring.io/spring-boot/docs/2.1.5.RELEASE/reference/html/using-boot-devtools.html). 
+To be able to see the changes in the browser tab, the page still needs to be reloaded. 
+That can also  be automated via a LiveReload browser extension. 
+One such extension for Google Chrome is [RemoteLiveReload](https://chrome.google.com/webstore/detail/remotelivereload/jlppknnillhjgiengoigajegdpieppei?hl=en-GB).
```
Comment (samie) [inline README.md]: I've been using LiveReload: https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en

Not sure, but is there a difference? Should we mention alternatives?

Comment (mehdi-vaadin) [reply README.md]: I used the one you've been using. I think one is enough. They can find alternatives by searching for "Live Reload".

File: README.md (hunk 1 above)
Comment (mehdi-vaadin) [reply README.md]: Done.

**Pull request merged**